CHANGELOG.MD

```diff
@@ -9,6 +9,9 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 :x: - deprecation
 :pencil: - chore
 
+## 2.0.1
+- :pencil: adjust dependencies
```

Comment:
rocket :)

Reply:
To me it is a bugfix rather, but let it be a 🚀